lib/alp_workloads/kvm_workload_utils.pm

```diff
@@ -23,6 +23,7 @@ use virt_autotest::utils qw(ssh_setup);
 our @EXPORT = qw(
   set_ct_engine
   set_kvm_container_image
+  set_host_bridge_interface_with_host
```

Comment:
This function seems more related with alp host setup, not kvm container. Suggest to call this function in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/virt_autotest/prepare_alp_host.pm#L25.